Added username rules to the page

EstebanMz opened the pull request:
Solves issue #44 
(Warning: Untested code)

EstebanMz committed:
Added username rules to the page

BGMP committed:
Merge branch 'master' of https://github.com/Re-Volt-America/Website into username-rules

BGMP commented on _i18n/strings.yml:
```diff
@@ -64,7 +64,13 @@ rules:
-      title: "B. Races & Scoring"
+      title: "B. Create an User Name"
+      1: 
+        title: "To play on RVA and be indexed in our official results, players must choose a nickname which matches the following rules:"
```
```suggestion
        title: "To play on RVA and be indexed in our official rankings, players must choose a nickname which matches the following rules:"
```

BGMP commented on _i18n/strings.yml:
```diff
@@ -64,7 +64,13 @@ rules:
+      title: "B. Create an User Name"
+      1: 
+        title: "To play on RVA and be indexed in our official results, players must choose a nickname which matches the following rules:"
+        bullet1: "Your nickname must have 16 characters at maximum."
```
```suggestion
        bullet1: "Your nickname must have from 1 to 16 characters."
```

BGMP commented on _i18n/strings.yml:
```diff
@@ -64,7 +64,13 @@ rules:
+      1: 
+        title: "To play on RVA and be indexed in our official results, players must choose a nickname which matches the following rules:"
+        bullet1: "Your nickname must have 16 characters at maximum."
+        bullet2: "It can include any characters from A-Z (uppercase or lowercase), numbers, underscore and space characters."
```
```suggestion
        bullet2: "Your nickname may contain any characters from A-Z (uppercase or lowercase), numbers, underscores and blank spaces"
```

BGMP reviewed (changes requested):
I've requested a couple of changes. You may apply them by clicking on "Commit Suggestion" if I'm not mistaken.

This commit breaks the website because the rest of the translation files at the i18n directory have not been updated with the new structure of `strings.yml`. I'll look into it tomorrow, for now just make sure to read my review.

Thanks Esteban!

EstebanMz committed:
Update _i18n/strings.yml

Co-authored-by: José Benavente <jose@bgm.cl>

EstebanMz committed:
Update _i18n/strings.yml

Co-authored-by: José Benavente <jose@bgm.cl>

EstebanMz committed:
Update _i18n/strings.yml

Co-authored-by: José Benavente <jose@bgm.cl>

BGMP committed:
Merge branch 'master' of https://github.com/Re-Volt-America/Website into username-rules

BGMP committed:
Fix translation files & add another rule

- Fixed translation keys in all languages.
- Added a bullet point regarding username uniqueness.

BGMP committed:
Formatting

BGMP reviewed (approved):
This looks good to go now. I fixed and added a couple of things here and there, otherwise the website wouldn't build. Thanks again!
